main.py

```diff
@@ -43,7 +47,7 @@ async def on_message(self, message):
         elif "!tacoscores" in message.content:
             data = get_this_months_scores(self.conn)
             data = [f"<@{taco[0]}> - {taco[1]}" for taco in data]
-            now = datetime.datetime.now()
+            now = datetime.now()
```

Comment:
This line is changed but the import statement at the top of the file is still "import datetime".

>>> import datetime
>>> datetime.now
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'datetime' has no attribute 'now'
>>> 

Either restore the line or change the import at the top to "from datetime import datetime"